hr_employee_social_media/views/hr_employee_view.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form" />
         <field name="arch" type="xml">
-            <xpath expr="//page[@name='public']/group" position="inside">
+            <xpath expr="//div[@id='o_work_employee_main']" position="inside">
```

Comment:
The previous place is no longer there?

Reply:
The previous place is change group to div.
I don't know which is better pattern between
`<xpath expr="//page[@name='public']/div/div" position="inside">`
and
`<xpath expr="//div[@id='o_work_employee_main']" position="inside">`